x/distribution/keeper/genesis.go

```diff
@@ -0,0 +1,50 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+// Get the set of all validator-distribution-info's with no limits, used during genesis dump
+func (k Keeper) GetAllVDIs(ctx sdk.Context) (vdis []types.ValidatorDistInfo) {
+	store := ctx.KVStore(k.storeKey)
+	iterator := sdk.KVStorePrefixIterator(store, ValidatorDistInfoKey)
+	defer iterator.Close()
+
+	for ; iterator.Valid(); iterator.Next() {
+		var vdi types.ValidatorDistInfo
+		k.cdc.MustUnmarshalBinary(iterator.Value(), &vdi)
+		vdis = append(vdis, vdi)
+	}
+	return vdis
+}
+
+// Get the set of all delegator-distribution-info's with no limits, used during genesis dump
+func (k Keeper) GetAllDDIs(ctx sdk.Context) (ddis []types.DelegationDistInfo) {
```

Comment:
I'd prefer if we didn't use acronyms for this, but personal preference.

Reply:
Yeah I don't mind changing